lms/static/scripts/frontend_apps/components/EmailPreferencesApp.tsx

```diff
@@ -42,6 +74,7 @@ export default function EmailPreferencesApp() {
             variant="custom"
             underline="always"
             href="https://web.hypothes.is/help/"
+            target="_blank"
```

Comment:
This change is unrelated to this PR, but I took the opportunity to add this missing `target"_blank".

Since the link goes to our own site, I thought it was safe to not add `rel="noopener noreferer"`.